### ISSUE TITLE
The profile picture frame constantly shows a notification.

### ISSUE THREAD

**Issue by 0xf581d2** [SpotX-Official/SpotX#609]
### 🧐 Have you read the FAQ and viewed similar issues?

- [X] I have read the FAQ
- [X] I have checked older issues, open and closed
- [X] I did a clean install of SpotX as <a href="https://telegra.ph/SpotX-FAQ-09-19#1.-Clean-installation">described in the FAQ</a>

### Сountry of your account

USA

### What is your Spotify Plan?

Free

### Do you have Spicetify installed?

No

### ℹ Computer information

```markdown
- Spotify: 1.2.42.290.g242057a2
- Windows: Windows 10 Enterprise LTSC 2021 21H2 19044.4651
- PowerShell: PowerShell 7 7.4.4 (Administrator)
```

### 📝 Description

- The profile picture frame always shows a blue notification dot, but when I click on it, there are no notifications (I have **subscribed to many podcasts**).

### 📸 Screenshots

![image](https://github.com/user-attachments/assets/720d25ca-883a-4626-ac0c-9613543870e7)

**Comment by amd64fox** [comment]
Usually, this notification disappears after opening the notification center and reloading the client (`Ctrl+Shift+R`)

but you have 0 notifications. If it doesn't work, give me your profile, and I'll send you a notification so you can try again.

**Comment by 0xf581d2** [comment]
https://open.spotify.com/user/ani8w5cef90wwzgkue2wu226i

**Comment by amd64fox** [comment]
I sent it, try again to do what is described above

**Comment by 0xf581d2** [comment]
resolved! thank you!

**Comment by SamBecca16** [comment]
> Usually, this notification disappears after opening the notification center and reloading the client (`Ctrl+Shift+R`)
> 
> but you have 0 notifications. If it doesn't work, give me your profile, and I'll send you a notification so you can try again.

I also got this issue, I did a clean install but still got the issue. My friend gave me a notification, and I try open it and reloading but the problem persists
Sorry for the inconvenience

**Comment by pWoLiAn** [comment]
I have the same issue too. Could you send me a notification as well @amd64fox or tell me how to do it myself

**Comment by amd64fox** [comment]
> Could you send me a notification as well

link ?

**Comment by pWoLiAn** [comment]
https://open.spotify.com/user/31yryiyophsah5tklv46vusatike?si=59d95d3992d54aad

**Comment by amd64fox** [comment]
sent

**Comment by pWoLiAn** [comment]
how do I open the notification console? When I click on the notification, it takes me to your profile.

**Comment by amd64fox** [comment]
you just need to open the notification center as in the screenshot above, then restart the client

**Comment by pWoLiAn** [comment]
![image](https://github.com/user-attachments/assets/4220444f-def3-4bb9-aa44-9a4269a29108)
Even after reloading, the notification is not getting cleared and the blue dot is still there. I even tried logging out and back in.
```
Spotify for Windows (64 bit)
1.2.45.454.gc16ec9f6
```
![image](https://github.com/user-attachments/assets/a9725cf5-f4bd-46d5-a4d0-be8476cb651f)
Another thing I noticed is that friends list is blank. But I'm not sure if this has anything to do with SpotX since it was blank even before I patched it.
